screensaver.digitalclock/addon.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="screensaver.digitalclock"
+	name="Digital Clock Screensaver"
+	version="6.0.0"
```

Comment:
We also have the option to use `5.1.0+matrix.1` so you can keep the versions more alike.

Reply:
If it's not an issue I'd like to keep things as they are.
I can barely keep track of changes in Kodi and somewhat lack of documentation. 
So I do things as I'm able to in order to get this screensaver to Kodi users. 
Unfortunately I'm not a developer, and all of this is extremely complicated for me.

Reply:
I think keeping the versions equal and just append `+matrix.1` will make your life easier?

Reply:
I believe in the guide text it used to say that we should have diffrent versions for different branches.
That's why I bumped this up to 6.0.0 for Matrix.
Wasn't planning on updating it for older branches but ended up doing it anyways.
So for now this is easier, we'll see moving forward how things will develop.